modules/client/src/channelProvider.ts

```diff
@@ -134,12 +187,12 @@ export class ChannelProvider {
 
   ///////////////////////////////////////////////
   ///// LISTENER METHODS
-  public on = (event: string, listener: (...args: any[]) => void): RpcConnection => {
+  public on = (event: string, listener: (...args: any[]) => void): any => {
     this.connection.on(event, listener);
     return this.connection;
   };
 
-  public once = (event: string, listener: (...args: any[]) => void): RpcConnection => {
+  public once = (event: string, listener: (...args: any[]) => void): any => {
```

Comment:
we can do a better job of the `event` and `return` types here as well probably

Reply:
Actually this could just be void

Reply:
Actually it can't because it extends from EventEmitter